test/test_char.py

```diff
@@ -43,6 +44,12 @@ def walk_tree(ast, parent=[]):
 @helpers.filtered_test
 def test_char(implementation):
     errors = []
+    # pyparser's fake_libc does not support the Arm headers
+    if 'supported_platforms' in implementation.metadata():
+        for platform in implementation.metadata()['supported_platforms']:
+            if platform['architecture'] == "arm_8":
+                return
```

Comment:
we've been doing `raise unittest.SkipTest()` instead of `return` but otherwise it looks good.

Reply:
good catch, thanks!